php-frontend/src/test/java/org/sonar/php/cfg/Validator.java

```diff
@@ -33,43 +36,95 @@ private Validator() {
     // this is an utility class and should not be instantiated
   }
 
-  static void assertCfgStructure(Map<String, BlockMetadata> data, String dotNotation, boolean testPredecessors) {
+  static void assertCfgStructureWithPredecessors(ControlFlowGraph cfg, Map<String, ExpectedBlockStructure> expectedStructure, int numberOfBlocks) {
```

Comment:
plural for `expectedStructure`? (while I prefer class renamed also)

Reply:
Well, the `ExpectedBlockStructure` contains the structure for a `CfgBlock`, while `expectedStructure` contains the structure for the whole CFG.

I'll update the names.